src/rules/prefer-in-document.js

```diff
@@ -32,8 +32,10 @@ function isAntonymMatcher(matcherNode, matcherArguments) {
 }
 
 function usesToBeOrToEqualWithNull(matcherNode, matcherArguments) {
-  return (matcherNode.name === "toBe" || matcherNode.name === "toEqual") &&
-    matcherArguments[0].value === null;
+  return (
```

Comment:
This is a result of the pre-commit linting tools; I've not made any changes to this code.